### DISCUSSION TITLE
plays/8c3afca8-9a3e-41b3-947a-89dd93898a99

### DISCUSSION THREAD

**Discussion by giscus** [reactplay/react-play#431] [Play Feedback]
# plays/8c3afca8-9a3e-41b3-947a-89dd93898a99

Developer jokes is a app where a random developer joke pops up every time you open it  or  click next joke

https://reactplay.io/plays/8c3afca8-9a3e-41b3-947a-89dd93898a99

**Comment by koustov** [comment]
This is a brilliant play. Most entertaining one.

**Reply by yung-coder** [reply]
Thanks 🙏

**Comment by Shivam-Katare** [comment]
This is such a great play. Love it. 😃

**Reply by yung-coder** [reply]
Thanks
